tests/test_glue/test_glue.py

```diff
@@ -9,19 +12,82 @@
 
 @mock_glue
 def test_create_job():
-    client = boto3.client("glue", region_name="us-east-1")
+    client = create_glue_client()
+    job_name = str(uuid4())
     response = client.create_job(
-        Name="test_name", Role="test_role", Command=dict(Name="test_command")
+        Name=job_name, Role="test_role", Command=dict(Name="test_command")
     )
-    assert response["Name"] == "test_name"
+    assert response["Name"] == job_name
 
 
 @mock_glue
 def test_create_job_default_argument_not_provided():
-    client = boto3.client("glue", region_name="us-east-1")
+    client = create_glue_client()
     with pytest.raises(ParamValidationError) as exc:
         client.create_job(Role="test_role", Command=dict(Name="test_command"))
 
     exc.value.kwargs["report"].should.equal(
         'Missing required parameter in input: "Name"'
     )
+
+
+@mock_glue
+def test_list_jobs():
+    client = create_glue_client()
+    expected_jobs = randint(1, 15)
+    for _ in range(expected_jobs):
+        create_test_job(client)
+    response = client.list_jobs()
+    response["JobNames"].should.have.length_of(expected_jobs)
```

Comment:
Following from an earlier comment - we should check here that the NextToken is not returned if all jobs are returned.
`response.shouldnt.have.key("NextToken")`